.github/workflows/mingw-w64.yml

```diff
@@ -66,34 +61,37 @@ jobs:
             mingw-w64-ucrt-x86_64-llvm
             mingw-w64-ucrt-x86_64-libffi
 
+      - name: Disable CRLF line ending substitution
+        run: |
+          git config --global core.autocrlf false

```

Comment:
If this is needed, perhaps `.gitattributes` should be updated in the crystal repo to fix the line endings on a per-filetype basis.

Reply:
[It's already there](https://github.com/crystal-lang/crystal/blob/master/.gitattributes), this is done to err on the side of caution like all other Windows workflows

Reply:
Ah, I didn't realise. If the others are like this, let's keep it but I think it'd be nice to turn it off to get CI warnings when gitattributes is wrong.